deploy/acm/crds/action.open-cluster-management.io_managedclusteractions.yaml

```diff
@@ -0,0 +1,121 @@
+
+---
```

Comment:
No need to include separator at the top of the file. Technically it makes it appear that there are 2 sections here, one empty and one with the CRD.